tools/auto_tuner/gen/rcar.json

```diff
@@ -0,0 +1,31 @@
+{"local":[
+    {
```

Comment:
clarify the names from here: 
https://github.com/codeplaysoftware/sycl-blas/blob/269a579928128c036729f8c90a60feb262c14dc1/include/operations/blas3_trees.h#L91